x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/kibana/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+import {
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiCallOut,
+  EuiText
+} from '@elastic/eui';
+import { Monospace } from '../components/monospace';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { statusTitle } from './common_kibana_instructions';
+
+export function getKibanaInstructionsForEnablingMetricbeat(product, _meta, {
+  esMonitoringUrl,
+  hasCheckedStatus,
+  checkingMigrationStatus,
+  checkForMigrationStatus,
+  autoCheckIntervalInMs
+}) {
+  const installMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.installMetricbeatTitle', {
+      defaultMessage: 'Install Metricbeat on the same server as Kibana'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.kibanaInstructions.installMetricbeatDescription"
+            defaultMessage="Follow {link}."
+            values={{
+              link: (
+                <EuiLink href="https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html" target="_blank">
+                  <FormattedMessage
+                    id="xpack.monitoring.metricbeatMigration.kibanaInstructions.installMetricbeatLinkText"
+                    defaultMessage="the instructions here"
+                  />
+                </EuiLink>
+              )
+            }}
+          />
+        </p>
+      </EuiText>
+    )
+  };
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Kibana module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+          metricbeat modules enable kibana-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.kibanaInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect Kibana monitoring metrics from http://localhost:5601. If the local Kibana instance has a different address, you must specify it via the {hosts} setting in the {file} file."
+              values={{
+                hosts: (
+                  <Monospace>hosts</Monospace>
+                ),
+                file: (
+                  <Monospace>modules.d/kibana-xpack.yml</Monospace>
```

Comment:
Per https://www.elastic.co/guide/en/kibana/master/monitoring-metricbeat.html (step 6) there are extra things you need to do if security is enabled.